internal/discover/discover.go

```diff
@@ -86,17 +86,10 @@ func (d *Discoverer) StartLocal(localPort int, localMCAddr string) {
 }
 
 func (d *Discoverer) startLocalIPv4Broadcasts(localPort int) {
-	bb, err := beacon.NewBroadcast(localPort)
-	if err != nil {
-		if debug {
-			l.Debugln("discover: Start local v4:", err)
-		}
-		l.Infoln("Local discovery over IPv4 unavailable")
```

Comment:
There's another log line somewhere that complains about no local discovery at all when both v4 and v6 fails to initialize - that'll never happen in this setup since we'll always add the v4 discovery thing and it'll retry continuously in the background... I think we can sacrifice that, but it should maybe be cleaned up...